ExampleMod/Localization/en-US.hjson

```diff
@@ -186,5 +186,10 @@ Mods: {
 			ExampleTip2: "This is yet another tip added by Example Mod!"
 			DisabledExampleTip: "This tip will be added but then disabled by Example Mod."
 		}
+
+		#This is used for shop conditions.
+		ShopConditions: {
+			RedPotCondition: "Describe what your condition is!"
```

Comment:
Proper localization please. `"When not happy enough or not a happy windy day"` or something...